apps/condo/pages/property/[id]/index.tsx

```diff
@@ -63,21 +72,69 @@ const PropertyInfoPanel: React.FC<IPropertyInfoPanelProps> = ({ title, message,
     </FocusContainer>
 )
 
+const PropertyListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 40px;
+`
+
+const PropertyReportCardBottomWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+  margin-top: 36px;
+  
+  & > div {
+    max-width: 50%;
+  }
+  
+  @media screen and (max-width: 550px) {
```

Comment:
I would prefer multiple antd spaces, since it's more clear to use and configure. `max-width` is not the best solution, when you have side-menu with dynamic width...That's why we're using something like `useContainerSize` in `@open-condo/ui`